lib/react.js

```diff
@@ -6632,7 +6610,7 @@ module.exports = ReactPropTypesSecret;
 /***/ (function(module, exports, __webpack_require__) {
 
 "use strict";
-/** @license React v16.8.6
+/** @license React v16.13.1
```

Comment:
Its probably nothing - but are we sure that the `react-is.development.js` that is getting bundled is kosher with version 17.x?

Reply:
That's a good question - I'm double checking this with @kealjones-wk!

Reply:
it appears that there [17.0.2 release of react-is](https://github.com/facebook/react/blob/master/packages/react-is/package.json) so id assume its good as long as when we upgrade to react 17 we are sure all the other react packages get updated to the latest 17 release.

Reply:
Cool cool, I double checked the react related dependencies to make sure they're all the latest. Those changes are in 4750550!